.github/workflows/pr.yaml

```diff
@@ -0,0 +1,15 @@
+name: Publish Extension
+on: pull_request
```

Comment:
I'm using this PR as a guinea pig to test the new "Load from URL" feature.

This will allow clients to test changes via early access. For this PR, the changes will be available to: `https://samepage.network/releases/dvargas92495/roamjs-query-builder/full-screen-mode`

Reply:
Convention on `main`: `https://samepage.network/releases/[owner]/[repo]`

Convention on PRs: `https://samepage.network/releases/[owner]/[repo]/[branch]`

Reply:
Once the PR is merged, should `https://samepage.network/releases/[owner]/[repo]/[branch]` still work?
I just tried https://samepage.network/releases/dvargas92495/roamjs-query-builder/full-screen-mode but got `Invalid URL`
https://samepage.network/releases/dvargas92495/roamjs-query-builder worked

Reply:
I misunderstood the Github docs and pointed it to the wrong environment variable: https://github.com/dvargas92495/roamjs-query-builder/actions/runs/4930733661/jobs/8812004090#step:4:12

The next version of `@samepage/scripts` will have this fixed